### ISSUE TITLE
How can I update the Zork recovery image version if I am already using ChromeOS in single boot

### ISSUE THREAD

**Issue by Anuragisme** [sebanc/brunch#1512]

**Comment by tejasraman** [comment]
Use the PWA

**Comment by tejasraman** [comment]
There are instructions on the troubleshooting page but if you want but I can provide a video

**Comment by Anuragisme** [comment]
Please provide the video.

**Comment by tejasraman** [comment]
Ok..... i'll try to make my existing brunch install do the trick. 

I'll send a video on the installation and usage of the PWA as a way to update Brunch, but it is best (for me at least) to use the enable_updates brunch option and use settings to update. I will edit this comment and add the video in about ½ an hour.

**Comment by tejasraman** [comment]
** UPDATE ** - Here is a video 

https://user-images.githubusercontent.com/96961804/163502788-cfe07d96-768c-4f71-be95-a15808a2b464.mp4

**Comment by Anuragisme** [comment]
Thanks a lot !!! for helping me.

**Comment by tejasraman** [comment]
Did it work?

**Comment by Anuragisme** [comment]
Yes it worked. Now I am getting updates.

**Comment by Anuragisme** [comment]
Could you send me that video again. I want enable updates on chrome os after reinstalling it. Please !!!

**Comment by tejasraman** [comment]
Yeah, here is the video
> Chromebook.video.mp4

**Comment by tejasraman** [comment]
Here:

https://user-images.githubusercontent.com/96961804/163502788-cfe07d96-768c-4f71-be95-a15808a2b464.mp4

**Comment by Anuragisme** [comment]
Thank you very much.

**Comment by tejasraman** [comment]
Anytime!

**Comment by tejasraman** [comment]
I can’t really get any info. In ChromeOS (settings), enable verbose boot when asked. Take a slow-motion video of the the onscreen text.

Also, what is our laptop model?

**Comment by Anuragisme** [comment]
But How to get into setting. Chrome OS is not starting.

Laptop Model : Lenovo V15 ADA
Processor : AMD Athalon Silver 3050U

**Comment by Anuragisme** [comment]
Yes, I have done that. What's the next step? I am sharing the video with
you.


https://photos.app.goo.gl/Nmm1tLCGBh8Y5L8V8

**Comment by Anuragisme** [comment]
This problem had occurred twice when I updated Chrome OS. and when i
reinstalled Chrome OS and updated it then happened the same thing.

**Comment by tejasraman** [comment]
2 things:
- Try kernel 5.10
- Try `pci=noafb` as a boot arg in cmdline boot args

**Comment by tejasraman** [comment]
ohhhhh, try updating brunch first to the **stable release**

**Comment by Anuragisme** [comment]
Yes I have tried updating Chrome OS after updating brunch to 102

**Comment by tejasraman** [comment]
pci=noafb? Kernel 5.15?
These are some fixes that might help you

**Comment by Anuragisme** [comment]
I tried them but not working. 😞😞😞

**Comment by Anuragisme** [comment]
Yes I tried all  that you told me. Yes I am on Zork

**Comment by tejasraman** [comment]
Try using the chromeos 101 recovery image with brunch 102 please? Send me a verbose after that pls?

**Comment by Anuragisme** [comment]
Sure, 'll send soon.

**Comment by Anuragisme** [comment]
Is there any way to do this other than reinstalling Chrome OS.

**Comment by tejasraman** [comment]
thanks for being prompt about that...
If you want, I'll run a test with brunch 102 on my end... 
Before that, try "nomodeset" as a kerel option

**Comment by Anuragisme** [comment]
Ok

**Comment by tejasraman** [comment]
Thanks for responding in <30 seconds! ;)

**Comment by Anuragisme** [comment]
😆😆😆

**Comment by Anuragisme** [comment]
Thanks you so much for you assistance.

**Comment by tejasraman** [comment]
did it work?

**Comment by Anuragisme** [comment]
Doing

**Comment by Anuragisme** [comment]
Please wait

**Comment by Anuragisme** [comment]
No it didn't work.

**Comment by tejasraman** [comment]
I've reached the end of my abilities. @sebanc do you know?

**Comment by Anuragisme** [comment]
No

**Comment by tejasraman** [comment]
what?

**Comment by Anuragisme** [comment]
I don't know about @sebanc

**Comment by tejasraman** [comment]
He is the main brunch developer

**Comment by Anuragisme** [comment]
How can I contact him ?

**Comment by tejasraman** [comment]
the @ mention sends a notification to his feed. He might look at the issue? Just wait a bit.

**Comment by Anuragisme** [comment]
Ok

**Comment by Anuragisme** [comment]
What are you doing.

**Comment by tejasraman** [comment]
In the menu, select ChromeOS (Settings) underneath "ChromeOS" with the down arrow and press "ENTER", select the normal options and type "YES" at verbose boot.

I can make another video if needed ;)
